compiler/rustc_typeck/src/coherence/inherent_impls.rs

```diff
@@ -49,13 +49,61 @@ impl ItemLikeVisitor<'v> for InherentCollect<'tcx> {
 
         let self_ty = self.tcx.type_of(item.def_id);
         let lang_items = self.tcx.lang_items();
+        let error_def_id = self.tcx.require_lang_item(hir::LangItem::Error, None);
+        let send_def_id = self.tcx.require_lang_item(hir::LangItem::Send, None);
+        let sync_def_id = self.tcx.require_lang_item(hir::LangItem::Sync, None);
```

Comment:
Can you move these into the `Dynamic` case or make them optional? That would avoid having to change `#![no_core]` tests.

Reply:
oh, absolutely, thank you for pointing that out!